docs/guides/language-packs.md

```diff
@@ -1,10 +1,96 @@
 # Language Packs
 
-A Language Pack is the mechanism by which the Functions binary can be extended to support additional runtimes, function signatures, even operating systems and installed tooling for a function. A Language Pack includes
-- a .builders.yaml file containing a reference to a builder OCI image reference, which conforms to the buildpack builder specification, and contains references to buildpacks supporting this Language Pack
-- one or more template directories illustrating the Language Pack's recognized function signatures
-- tests and documentation for the templates
+A Language Pack is the mechanism by which the Functions binary can be extended to support additional runtimes, function signatures, even operating systems and installed tooling for a function. A Language Pack is a directory, typically named for the language or runtime being templated, and includes
 
-Built in to the Functions library is a basic language pack for each supported language.
+- one or more directories containing templates for the Language Pack's recognized function signatures
+- each of which contains a `manifest.yaml` file with metadata about the template
+- tests and documentation
 
-For an example external language pack, see [https://github.com/lance/gcf-kn/tree/main/ruby]
+For example, a Language Pack directory for Ruby with templates for both
+a CloudEvent function signature and an HTTP function signature, may look
+similar to the following directory tree.
+
+```
+ruby
+├── cloudevent
+│   ├── func.rb
+│   ├── Gemfile
+│   ├── manifest.yaml
+│   ├── Rakefile
+│   ├── README.md
+│   └── test.rb
+└── http
+    ├── func.rb
+    ├── Gemfile
+    ├── manifest.yaml
+    ├── Rakefile
+    ├── README.md
+    └── test.rb
+```
+
+## `manifest.yaml`
```

Comment:
I thought that `manifest.yaml` would be placed at the Language Pack Repository Root.  Basically, at the URL which is provided to `func repository add`.  If func searches for a manifest.yaml at that URL (the repository's manifest), then the developer of the Language Pack would be free to place all their runtimes in a subdirectory.  Are we talking about the same manifest.yaml?